README.md

```diff
@@ -144,15 +108,8 @@ make lint
 
 When you're ready to submit a pull request, make sure you've removed anything that doesn't seem to belong (`go.mod`/`go.sum` changes, content from pulumi/docs, etc.) and submit the PR in the usual way.
 
-If you're doing work in another repository that's associated with the changes in your PR, you can "pin" your PR branch to another module repository branch by pointing Hugo to that branch. To do that, use `hugo mod get` and pass a reference to the target branch:
-
-```
-hugo mod get github.com/pulumi/pulumi-hugo@my-special-branch
-```
-
-This will modify `go.mod` and `go.sum` accordingly and result in a PR preview that incorporates your changes from the other branch. Just be sure to remove these changes before you're ready to merge.
-
-If you would like to generate API docs for packages beyond AWS and Aiven (the docs available in PR previews by default), add the package name you would like docs for [in this file](https://github.com/pulumi/registry/blob/master/scripts/ci/build.sh#L5). Be sure to remove these changes before merging.
+If you would like to generate API docs for packages beyond AWS and Aiven (the docs available in PR previews by default), add the package name you would like docs for [in this file](https://github.com/pulumi/registry/blob/master/scripts/ci/build.sh#L5). Be sure to remove these changes before merging. 
```

Comment:
Observationally, this is no longer true. I don't see any mention of AWS or Aiven in `scripts/ci/build.sh`, and the registry builds all docs packages by in PR reviews:

For example (from this PR as I review it):
http://registry--origin-pr-5876-98aa61ae.s3-website.us-west-2.amazonaws.com/registry/packages/aws-native/api-docs/apigatewayv2/authorizer/

Reply:
thank you I should have removed this one also. It is showing up as something new because of the way it diffs. yes we do generate all packages on builds now.